docs/redirects.js

```diff
@@ -280,32 +280,48 @@ const redirects = [
     to: "/articles/templates",
   },
   {
-    from: ["/testkube-cloud/intro", "/testkube-cloud"],
-    to: "/testkube-cloud/articles/intro",
+    from: ["/testkube-cloud/intro", "/testkube-cloud", "/testkube-cloud/articles/intro"],
+    to: "/testkube-pro/articles/intro",
```

Comment:
Shouldn't these `from` also include the short version for `testkube-pro`?
```suggestion
    from: ["/testkube-cloud/intro", "/testkube-cloud", "/testkube-cloud/articles/intro", "/testkube-pro/intro", "/testkube-pro"],
    to: "/testkube-pro/articles/intro",
```

Reply:
good catch, updated!